api/database/migrations/2023_10_23_195252_create_assessment_results_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Query\Expression;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('assessment_results', function (Blueprint $table) {
+            $table->uuid('id')->primary('id')->default(new Expression('gen_random_uuid()'));
+            $table->timestamps();
+            // belongsTo three tables, the third isn't required
+            $table->uuid('assessment_step_id');
+            $table->foreign('assessment_step_id')->references('id')->on('assessment_steps')->cascadeOnDelete(true);
+            $table->uuid('pool_candidate_id');
+            $table->foreign('pool_candidate_id')->references('id')->on('pool_candidates')->cascadeOnDelete(true);
+            $table->uuid('pool_skill_id')->nullable();
+            $table->foreign('pool_skill_id')->references('id')->on('pool_skill')->cascadeOnDelete(true);
```

Comment:
I don't think I'd cascade this one either, for the same reason.

Reply:
Changed in https://github.com/GCTC-NTGC/gc-digital-talent/commit/3460f2b10de4d38149d677f49a3fc53ebb7549d3